main/timeseries.py

```diff
@@ -122,6 +128,28 @@ def get_charged_effort_timeseries(
     return timeseries
 
 
+def get_active_team_members(start_date: datetime, end_date: datetime) -> int:
```

Comment:
Actually, the number of active members should be calculated as those having a capacity above zero for the time period. Chances are that everyone with a capacity above zero will be reporting something in the given month, but they are not necessarily the same thing.